hawkularclient.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency('shoulda')
   gem.add_development_dependency('rspec-rails', '~> 3.0')
   gem.add_development_dependency('rake', '< 11')
-  gem.add_development_dependency('simple-websocket-vcr', '= 0.0.4')
+  gem.add_development_dependency('simple-websocket-vcr', '= 0.0.7')
```

Comment:
because it uses the new feature from version `0.0.7` (https://github.com/Jiri-Kremser/simple-websocket-vcr/pull/6)